ivy_tests/test_ivy/helpers/testing_helpers.py

```diff
@@ -907,15 +907,15 @@ def _create_transpile_report(
     data: dict, backend: str, file_name: str, is_backend: bool = False
 ):
     if not is_backend:
-        backend_specific_data = ["backend_nodes", "frontend_time", "args", "kwargs"]
+        backend_specific_data = ["nodes", "time", "args", "kwargs"]
     else:
-        backend_specific_data = ["backend_nodes", "backend_time", "args", "kwargs"]
+        backend_specific_data = ["nodes", "time", "args", "kwargs"]
```

Comment:
Can remove this else statement now

Reply:
done

Reply:
I dont see the changes 🤔 
But you can remove both if and else statement and just add `backend_specific_data = ["nodes", "time", "args", "kwargs"]` instead of them